docs/source/first_steps/cps_ood.rst

```diff
@@ -0,0 +1,133 @@
+
+.. note::
+   If **Working Directory** left blank, the session will launch in the main class folder (in this example ``/work/cps/ALY3070``). Alternatively, start the session directly from your personal working directory by entering: ``/work/cps/ALY3070/students/[username]``, where ``[username]`` is your username on Discovery. The instructions below assume the field is left blank. 
+
+Wait until the session is successfully created and started (turns green).
+
+#. 5. Click **Connect to Jupyter** to open JupyterLab.
+ 
+.. image:: /images/cps-ood-jupyterlab-start-session.png
+ :width: 400
+ :alt: session created view. 
+
+This will open a JupyterLab interface in another tab.
+
+#. 6. Select **Cancel** when prompted with the **Build Recommended** option. 
+
+.. image:: /images/cps-ood-build-window.png
+ :width: 400
+ :alt: build window view.
+
+The package jupyterlab-dash does not require a build, and will not work when build is enabled.
+
+Step 2 - access your class directories
+=======================================
+After you are connected to a CPS JupyterLab session on OOD, you can access any shared class directories and your private class directory.
+
+#. 1. You can navigate between the class folders using the left menu. Your instructor may share files in this directory.
+
+.. image:: /images/cps-ood-jupyterlab-folders-view.png
+ :width: 400
+ :alt: show files.
+
+For instance, file **Example.ipynb** can be viewed using Python Jupyter Notebook (but not edited or removed).
+
+#. 2. Navigate to the **students** directory, where you will see another directory under your username.
+
+.. image:: /images/cps-ood-jupyterlab-students-folder.png
+ :width: 400
+ :alt: show students folder.
+
+Enter your personal class directory.
+
+.. image:: /images/cps-ood-jupyterlab-username-folder.png
+ :width: 400
+ :alt: show inside username folder.
+
+Here you can create and edit files. 
+
+#. 3. Open a new Python Notebook session from the Launcher menu by clicking the **Python 3 (ipykernel)**. 
+
+.. image:: /images/cps-ood-jupyterlab-ipykernel-launcher.png
+ :width: 400
+ :alt: show inside ipykernel.
+
+A new file will be created inside your directory called **Untitled.ipynb**. You can rename it by right-click + Rename option. 
+
+.. image:: /images/cps-ood-jupyterlab-ipykernel.png
+ :width: 400
+ :alt: show inside ipykernel.
+
+This Python notebook has ready-to-use Python packages needed for your class.
+
+.. note:: 
+  **Permission Denied errors:**
+  Do not attempt to create, edit or write files that are outside of your personal student directory. Most "Permission Denied" errors are due to directories or files having read-only access permissions. 
+
+Step 3 - submit class assignments
+=================================
+Due to the write-only access permissions on the **assignments** directory, it is required to use the command line interface (Linux Terminal) to submit assignments. **Using other methods currently does not work**.
```

Comment:
Currently, using other methods, such as OOD interface (?) does not work

Reply:
@marianalevineu looks like this hasn't been corrected (do->does)